vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -51,18 +51,17 @@ class MSBuildTools {
     newInfo(`Build configuration: ${buildType}`);
     newInfo(`Build platform: ${buildArch}`);
 
-    //const verbosityOption = verbose ? 'normal' : 'quiet';
-    const verbosityOption = 'normal';
+    const verbosityOption = verbose ? 'normal' : 'quiet';
     const args = [
       `/clp:NoSummary;NoItemAndPropertyList;Verbosity=${verbosityOption}`,
       '/nologo',
       `/p:Configuration=${buildType}`,
       `/p:Platform=${buildArch}`,
       '/p:AppxBundle=Never',
+      '/bl',
```

Comment:
Do we want to output a binlog by default? I don't like the idea of writing unnecessary files to the users project (guessing that's the cwd?) if we don't have to.

Reply:
we already were writing the binlog. This is invaluable for debugging build issues, both in the lab as well as locally, so I'd rather not remove it unless we have a strong reason to.

---
In reply to: [405941293](https://github.com/microsoft/react-native-windows/pull/4536#discussion_r405941293) [](ancestors = 405941293)

Reply:
What's the `cwd` here? Maybe we could output it as an arg? Dropping this in on the users machine on every run feels very annoying if we don't want it in the vast majority of cases.

Reply:
It seems fine if it goes into a working directory for the build for internals to RNW. Less so if it's in a user's app package.